inverse_test.go

```diff
@@ -89,11 +140,11 @@ func TestInverseBloomFilter_Encode(t *testing.T) {
 	}
 
 	if f.capacity != f2.capacity {
-		t.Errorf("Different capacities")
+		t.Error("Different capacities")
```

Comment:
i don't really like these generic errors. we should include actual + expected values in the error message